rest-api/src/main/scala/za/co/absa/enceladus/rest_api/repositories/DatasetMongoRepository.scala

```diff
@@ -77,13 +76,24 @@ class DatasetMongoRepository @Autowired()(mongoDb: MongoDatabase)
    * @return List of Menas references to Datasets, which contain the relevant conformance rules
    */
   def containsMappingRuleRefEqual(refColVal: (String, Any)*): Future[Seq[MenasReference]] = {
```

Comment:
Originally written by @lokm01, but some thing were missing (enabled check, sorting), but it it nicely general